pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -648,6 +705,37 @@ func (m *migrationMonitor) startMonitor() {
 	}
 }
 
+func (l *LibvirtDomainManager) getJobStatsWithRetry(dom cli.VirDomain) (*libvirt.DomainJobInfo, error) {
```

Comment:
this can be handled by the loop retry in the monitor loop now.

Reply:
yea, you're right. removed